index.ts

```diff
@@ -53,6 +53,12 @@ ic.call_raw128 = function (...args) {
 
 type ic = {
     accept_message: () => void;
+    /**
+     * Returns the argument data as bytes.
+     * @returns the argument data
+     */
+    arg_data_raw: () => blob;
+    arg_data_raw_size: () => nat64;
```

Comment:
Why no JS/TS doc here?

Reply:
I added it for arg_data_raw so we could have a discussion. I've removed it though and will wait until we do #470 to add them in consistenly.